internal/ext/importer_test.go

```diff
@@ -758,93 +754,21 @@ func TestImport_Rollouts_LTVersion1_1(t *testing.T) {
 	assert.EqualError(t, err, "flag.rollouts is supported in version >=1.1, found 1.0")
 }
 
-func TestImport_Namespaces(t *testing.T) {
-	tests := []struct {
-		name         string
-		docNamespace string
-		cliNamespace string
-		wantError    bool
-	}{
-		{
-			name:         "import with namespace",
```

Comment:
i think we still want some of these tests. I think we should still test:

- a file with a single namespace (non-stream)
- a file with no namespace (default namespace, non-stream)
- a stream file with multiple documents, but no namespace (should all go under default namespace)
- a stream file with multiple documents, under multiple namespaces (all unique, maybe one default)
- a stream file with multiple documents, multiple namespaces, some overlap (non-unique, what happens?)

are any of these covered by existing tests?

Reply:
@markphelps You are right about these different cases to at least give us that piece of mind. I wrote tests cases for the following: https://github.com/flipt-io/flipt/pull/2151/commits/c05f96f2f5ea6b68b4cef620b827e94e7b1915a7